### PR TITLE
fixed some stuff

### PR DESCRIPTION
**Pull request by jlam609**
fixed stuff

### CODE REVIEW

File: src/components/jobDetailsComponent/userButtons.tsx [hunk 1]
```diff
@@ -83,16 +83,14 @@ const UserButtons: React.FC = () => {
...
-        <Button variant="outlined" onClick={handleReserve}>
-          {job.reserved ? (
-            <CloseIcon className="buttonIcon" />
-          ) : (
-            <DoneIcon className="buttonIcon" />
-          )}
-          {job.reserved ? 'Cancel' : 'Reserve'}
-        </Button>
-      )}
+      <Button variant="outlined" onClick={handleReserve}>
```
Comment (bansara) [inline src/components/jobDetailsComponent/userButtons.tsx:86]: why did you change this? cancel and reserve should only show if the person seeing the page is the one who reserved the job.

**Review by bansara — CHANGES_REQUESTED** [review]
mostly fine, just one issue - see comment.

File: src/components/jobDetailsComponent/userButtons.tsx (hunk 1 above)
Comment (jlam609) [reply src/components/jobDetailsComponent/userButtons.tsx:86]: Oh there was a bug when we were first going through it that the button was never showing for people to reserve the job

**Review by bansara — APPROVED** [review]
